designer/server/src/main/scala/pl/touk/nussknacker/ui/api/NuDesignerOpenApi.scala

```diff
@@ -0,0 +1,21 @@
+package pl.touk.nussknacker.ui.api
+
+import pl.touk.nussknacker.ui.api.app.AppApiHttpService
+import sttp.tapir.server.ServerEndpoint
+import sttp.tapir.swagger.SwaggerUIOptions
+import sttp.tapir.swagger.bundle.SwaggerInterpreter
+
+import scala.concurrent.Future
+
+class NuDesignerOpenApi(appApiHttpService: AppApiHttpService) {
+
+  val publicServerEndpoints: List[ServerEndpoint[Any, Future]] =
+    SwaggerInterpreter(
+      swaggerUIOptions = SwaggerUIOptions.default.copy(pathPrefix = "api" :: "docs" :: Nil)
+    )
+      .fromEndpoints(
+        appApiHttpService.allEndpointDefinitions,
+        "Nussknacker Designer API",
+        "1.0.0"
```

Comment:
wait. The API is already in production :) We're just providing an OpenAPI document for it. 
I don't know how we treat the current API, but for sure it's not an alpha ;) 
Maybe we should discuss this with the team.

Reply:
I know that it is on production but exposing swagger definition suggests that it is for service integration purposes. So far it wasn't prepared for that

Reply:
I couldn't agree. We can have Swagger for internal API and the Nu Designer API is an internal one. Exposing Swagger should not suggest it's an API for external clients.

Because it's an internal API, IMO, it'd be nice to introduce SemVer and take advantage of its rules during development and maintaining contract between Frontend and Backend teams

Reply:
Let's talk f2f

Reply:
as we agreed: for now, there will be no API version